src/utils/AppDevUtils.ts

```diff
@@ -57,9 +59,21 @@ const validateURL = (url: string): boolean => {
   }
 };
 
+/* Sort matches by meetingTime in descending order */
+const sortMatchByMeetingTime = (x: Match, y: Match) => {
```

Comment:
Was wondering why we put this under AppDevUtils since these are matching specific functions

Reply:
Functions in the repo layer are for specifically interacting with the database. Makes more sense to have standard utility functions abstracted.